README-PI-DEBIAN.md

```diff
@@ -0,0 +1,57 @@
+### Building on Pi OS/Debian
+
```

Comment:
I'm not sure if we should call this document like that.
Most of the info here is generic Linux information.
The only specific info for Debian-based distibutions (Raspberry Pi OS, Ubuntu, Linux Mint, ...) are the `sudo apt install` commands and some wording.
The problem I see here, is that we now have duplicated documentation and need to maintain it in two places. Maybe it is better to include it in the main readme.md with some wording to explain that the commands are examples for Debian-based distributions.

Reply:
okay. I will try and merge it with the main readme